src/Workleap.DomainEventPropagation.Analyzers/EventDomainAttributeUsageAnalyzer.cs

```diff
@@ -18,8 +19,17 @@ public sealed class EventDomainAttributeUsageAnalyzer : DiagnosticAnalyzer
         isEnabledByDefault: true,
         helpLinkUri: RuleIdentifiers.HelpUri);
 
+    internal static readonly DiagnosticDescriptor UseUniqueNameAttribute = new(
```

Comment:
With target-typed new like here for these two diagnostic descriptors, [this .NET bug](https://github.com/dotnet/roslyn-analyzers/issues/5828) kicks in and display this error which seems to be fixed on newer versions of .NET. We must specify the full type name 🤷 

![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/14242083/1009efeb-905f-4bbf-998b-72e16e3f4286)